### PR TITLE
Use proxy in crwdstrike

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-37600

## Description
passing proxy param to generic http call

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_13_9.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### CrowdStrike Falcon
+
+- Added support for passing ***proxy*** arg in generic_http calls.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_13_9.md:6]:
```suggestion
- Added support for passing the *proxy* argument in generic_http calls.
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Review by DeanArbel — CHANGES_REQUESTED** [review]
Please remove existing `handle_proxy()` from the code where possible

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py [hunk 2]
```diff
@@ -386,6 +385,8 @@ def http_request(method, url_suffix, params=None, data=None, files=None, headers
             data=data,
             files=files,
             params=params,
+            proxy=PROXY,
+            prox
```
Comment (DeanArbel) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:389]: please remove

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**